app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Admins</h1>
+
+<p><%= link_to "Create new admin", new_registration_path %></p>
+
+<table>
+<% @admins.each do |admin| %>
+  <tr>
+    <td><%= admin.email %></td>
+    <td><%= link_to "Edit", edit_registration_path(admin) %></td>
```

Comment:
Coming soon! That's a feature in the upcoming PR that I'm going to issue after this one is merged, when permissions and roles are introduced.